image/store.go

```diff
@@ -67,21 +67,21 @@ func NewImageStore(fs StoreBackend, lss LayerGetReleaser) (Store, error) {
 
 func (is *store) restore() error {
 	err := is.fs.Walk(func(dgst digest.Digest) error {
-		img, err := is.Get(IDFromDigest(dgst))
+		img, err := is.Get(ID(dgst))
 		if err != nil {
-			logrus.Errorf("invalid image %v, %v", dgst, err)
+			logrus.WithField("digest", dgst).WithError(err).Error("invalid image")
 			return nil
 		}
 		var l layer.Layer
 		if chainID := img.RootFS.ChainID(); chainID != "" {
 			if !system.IsOSSupported(img.OperatingSystem()) {
-				logrus.Errorf("not restoring image with unsupported operating system %v, %v, %s", dgst, chainID, img.OperatingSystem())
+				logrus.WithField("digest", dgst).WithField("chainID", chainID).WithField("os", img.OperatingSystem()).Errorf("not restoring image with unsupported operating system")
```

Comment:
this one can be reduced to:
```suggestion
				log.Error("not restoring image with unsupported operating system")
```

Reply:
also, Errorf -> Error